src/QtUtils/MockCliProgram/main.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/numbers.h>
+
+#include <chrono>
+#include <iostream>
+#include <string_view>
+#include <thread>
+
+int main(int argc, char* argv[]) {
+  if (argc < 2) {
+    std::cout << "Some example output" << std::endl;
+    return 0;
+  }
+
+  if (std::string_view{argv[1]} == "--sleep_for_ms") {
```

Comment:
I know that this is just a fake for testing, but why not use abseil flags like in the rest of the codebase instead of having ad-hoc code for parsing flags?

Reply:
That was a good idea. Made the code way shorter. done